WorkManagerSample/app/src/main/java/com/example/background/FilterViewModel.kt

```diff
@@ -32,7 +32,7 @@ class FilterViewModel(application: Application) : ViewModel() {
     private val workManager = WorkManager.getInstance(application)
 
     internal val workInfo =
-        workManager.getWorkInfosByTagLiveData(Constants.TAG_OUTPUT).map { it[0] }
```

Comment:
OOC, why this change? Was it unclear before ?

Reply:
Encountered an issue when the list was empty. Changed it back to use `map` again.